src/Api.Rest/Contracts/RawDataServiceFeatureMatrix.cs

```diff
@@ -41,10 +41,18 @@ public RawDataServiceFeatureMatrix( [NotNull] InterfaceVersionRange interfaceVer
 
 		public static Version RawDataArchiveLookupMinVersion { get; } = new Version( SupportedMajorVersion, 5 );
 
+		public static Version UpdateRawDataInformationMinVersion { get; } = new Version( SupportedMajorVersion, 6 );

```

Comment:
Jetzt wo ich das sehe fällt mir auf das die Version so nicht stimmen kann, da der RawDataService 1.6 schon in PiWeb 8.0 drin ist. Du müsstest also im Backend noch die Version vom RawDataService  auf 1.7 erhöhen und hier auch auf 7 ändern, weil du ja ein neues Feature hinzugefügt hast.

Reply:
Ja, das habe ich vergessen. Die Version wurde angepasst